### PR TITLE
Design static pages

### PR DESCRIPTION
**Pull request by phucynwa**

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "style.scss";
+@import "font-awsome.scss";
```
Comment (devtuananh) [inline app/assets/stylesheets/custom.scss]: Dùng nháy đơn cho css, js

File: app/assets/javascripts/custom.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                $(image).css('animation-delay', delay);
+                $(image).addClass('animated flipInX');
+            });
+
+        },
+        template: ' <div class="col-md-4 col-sm-4 col-xs-4"><div class="insta-image"><a href="{{link}}" target="_blank"><img src="{{image}}" /><div class="likes">&hearts; {{likes}}</div></a></div></div>'
+    });
+    userFeed.run();
+});
+
```
Comment (devtuananh) [inline app/assets/javascripts/custom.js]: Xóa dòng thừa

File: app/assets/javascripts/custom.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+$(document).ready(function () {
+    $(window).scroll(function () {
```
Comment (huongnguyenmta) [inline app/assets/javascripts/custom.js]: check all indent in this file

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title)
```
Comment (huongnguyenmta) [inline app/helpers/application_helper.rb]: remove parentheses

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        <div class="footer-details">
+          <h4><%= t ".get_in_touch" %></h4>
+          <ul class="list-unstyled footer-contact-list">
+            <li>
+              <i class="fa fa-map-marker fa-lg"></i>
+              <p><%= t ".address" %></p>
+            </li>
+            <li>
+              <i class="fa fa-phone fa-lg"></i>
+              <p><a href="tel:<%= t ".phone" %>"><%= t ".phone" %></a></p>
```
Comment (huongnguyenmta) [inline app/views/layouts/_footer.html.erb]: what is this? ` href="tel:<%= t ".phone" %>"><%= t ".phone" %>`

Comment (phucynwa) [reply app/views/layouts/_footer.html.erb]: @huongnguyenmta It means "click to call".

**Pull request merged**